asf_online_data_exploration/config/data_collection_parameters.py

```diff
@@ -0,0 +1,193 @@
+"""
+Script with parameters for data collection.
+"""
+
+# -----TWITTER DATA COLLECTION-----
+
+heating_technologies_ruleset_twitter = [
+    {
+        "value": '"heat pump" OR "heat pumps" OR #heatpump OR #heatpumps',
+        "tag": "general_heat_pumps",
+    },
+    {
+        "value": '"air source hp" OR "air source hps" OR "air-source hp" OR "air-source hps" OR #ashp OR #airsourcehp OR #airsourcehps OR #airsourceheatpump OR #airsourceheatpumps',
+        "tag": "ashp",
+    },
+    {
+        "value": '"ground source hp" OR "ground source hps" OR "ground-source hp" OR "ground-source hps" OR #gshp OR #groundsourcehp OR #groundsourcehps OR #groundsourceheatpump OR #groundsourceheatpumps',
+        "tag": "gshp",
+    },
+    {
+        "value": '"water source hp" OR "water source hps" OR "water-source hp" OR "water-source hps" OR #wshp OR #watersourcehp OR #watersourcehps OR #watersourceheatpump OR #watersourceheatpumps',
+        "tag": "wshp",
+    },
+    {
+        "value": '"air to air hp" OR "air to air hps" OR "air-to-air hp" OR "air-to-air hps" OR "air2air hp" OR "air2air hps" OR "a2a hp" OR "a2a hps" OR #atahp OR #airtoairhp OR #airtoairhps OR #airtoairheatpump OR #airtoairheatpumps OR #a2ahp OR #a2ahps OR #air2airhp OR #air2airhps OR #air2airheatpump OR #air2airheatpumps',
+        "tag": "atahp",
+    },
+    {
+        "value": '"air to water hp" OR "air to water hps" OR "air-to-water hp" OR "air-to-water hps" OR "air2water hps" OR "air2water hp" OR "a2w hp" OR "a2w hps" OR #atwhp OR #airtowaterhp OR #airtowaterhps OR #airtowaterheatpump OR #airtowaterheatpumps OR #a2whp OR #a2whps OR #air2waterhp OR #air2waterhps OR #air2waterheatpump OR #air2waterheatpumps',
+        "tag": "atwhp",
+    },
+    {
+        "value": '"hybrid hp" OR "hybrid hps" OR "bivalent hp" OR "bivalent hps" OR "warm air hp" OR "warm air hps"',
+        "tag": "heat_pump_others",
+    },
+    {
+        "value": '"gas boiler" OR "gas boilers" OR #gasboiler OR #gasboilers',
+        "tag": "gas_boilers",
+    },
+    {
+        "value": '"oil boiler" OR "oil boilers" OR #oilboiler OR #oilboilers',
+        "tag": "oil_boilers",
+    },
+    {
+        "value": '"hydrogen boiler" OR "hydrogen boilers" OR "h2 boiler" OR "h2 boilers" OR #hydrogenboiler OR #hydrogenboiler OR #h2boiler OR #h2boilers',
```

Comment:
@ch-williamson, since we're now collecting data for "boiler", none of the "X type boiler" are now needed (only the hashtags), so will just make sure to make it a search term during the analysis :)